<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>FormGpio</class>
    <widget class="QWidget" name="FormGpio">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>845</width>
                <height>555</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>FormGpio</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
            <property name="leftMargin">
                <number>9</number>
            </property>
            <property name="topMargin">
                <number>9</number>
            </property>
            <property name="rightMargin">
                <number>9</number>
            </property>
            <property name="bottomMargin">
                <number>9</number>
            </property>
            <property name="horizontalSpacing">
                <number>6</number>
            </property>
            <item row="0" column="0">
                <widget class="QLabel" name="u_labelTitle">
                    <property name="text">
                        <string>GPIO</string>
                    </property>
                </widget>
            </item>
            <item row="1" column="0">
                <widget class="QSplitter" name="splitter">
                    <property name="orientation">
                        <enum>Qt::Vertical</enum>
                    </property>
                    <widget class="TableWidgetGpio" name="u_tableWidgetGpio">
                        <property name="selectionMode">
                            <enum>QAbstractItemView::SingleSelection</enum>
                        </property>
                        <property name="selectionBehavior">
                            <enum>QAbstractItemView::SelectRows</enum>
                        </property>
                        <attribute name="horizontalHeaderCascadingSectionResizes">
                            <bool>false</bool>
                        </attribute>
                        <attribute name="horizontalHeaderStretchLastSection">
                            <bool>true</bool>
                        </attribute>
                        <column>
                            <property name="text">
                                <string>引脚名</string>
                            </property>
                        </column>
                        <column>
                            <property name="text">
                                <string>信号</string>
                            </property>
                        </column>
                        <column>
                            <property name="text">
                                <string>输出电平</string>
                            </property>
                        </column>
                        <column>
                            <property name="text">
                                <string>模式</string>
                            </property>
                        </column>
                        <column>
                            <property name="text">
                                <string>上下拉</string>
                            </property>
                        </column>
                        <column>
                            <property name="text">
                                <string>速度</string>
                            </property>
                        </column>
                        <column>
                            <property name="text">
                                <string>用户标签</string>
                            </property>
                        </column>
                    </widget>
                    <widget class="QWidget" name="u_widgetPinAttribute" native="true">
                        <layout class="QGridLayout" name="u_gridLayoutPinAttribute">
                            <property name="leftMargin">
                                <number>0</number>
                            </property>
                            <property name="topMargin">
                                <number>0</number>
                            </property>
                            <property name="rightMargin">
                                <number>0</number>
                            </property>
                            <property name="bottomMargin">
                                <number>0</number>
                            </property>
                            <property name="spacing">
                                <number>0</number>
                            </property>
                        </layout>
                    </widget>
                </widget>
            </item>
        </layout>
    </widget>
    <customwidgets>
        <customwidget>
            <class>TableWidgetGpio</class>
            <extends>QTableWidget</extends>
            <header>TableWidgetGpio.h</header>
        </customwidget>
    </customwidgets>
    <resources/>
    <connections/>
</ui>
